src/plugins/intel_gpu/src/graph/impls/ocl/convolution.cpp

```diff
@@ -94,7 +94,7 @@ struct convolution_impl : typed_primitive_impl_ocl<convolution> {
             const auto spatial_rank = input_layout.get_spatial_rank();
 
             ov::PartialShape kernel;
-            for (size_t i = 0; i < spatial_rank; i++) {
+            for (int32_t i = (int32_t)spatial_rank - 1; i >= 0; i--) {
```

Comment:
```suggestion
            for (int32_t i = std::static_cast<int32_t>(spatial_rank) - 1; i >= 0; i--) {
```

Reply:
Edited as you commented. Thanks for your comment.